### PR TITLE
Some updates

### PR DESCRIPTION
**Pull request by jycouet**
Hi, 

Please find here some updates that I recommend.

I found also something new while doing this that I want to address ASAP.
Here is the link: https://github.com/jycouet/kitql/issues/79

### CODE REVIEW

File: src/routes/continents/[code]/index.svelte [hunk 1]
```diff
@@ -1,28 +1,40 @@
...
-	);
-
-	// console.log('continentForPatch:', continentForPatch);
-
-	KQL_AllCountriesOfContinent.patch({ continent: continentForPatch });
+	export async function load({ fetch, url, params, session, stuff }) {
+		// await KQL_AllCountriesOfContinent.query({ fetch, variables: { code: params.code } });
+		return {};
+	}
+</script>
```
Comment (frederikhors) [inline src/routes/continents/[code]/index.svelte]: Is this needed?

**Comment by frederikhors** [comment]
I think we should add this demo to your repository. I'll accept this PR after your answer and I will create a PR for your examples dir. Do you agree?

**Comment by jycouet** [comment]
I love tailwindcss, but I would like to avoid adding things to the demo that is not directly related. (Or maybe in the complex demo only)

I think that the best is to evolve Demo 1 with routes & patch examples shown here. 👍 (I don't see the point of having 2 demo VERY close to each other (I think about maintenance too ^^)).

I would like also to close: https://github.com/jycouet/kitql/issues/79 & https://github.com/jycouet/kitql/issues/80 before `Demo1` evolution.

Is that good with you?
I will suggest the Demo 1 evolution and let you validate it? (To make sure I didn't miss anything you wanted to see?)

**Comment by frederikhors** [comment]
I'll answer to you in a few minutes! You're great!!!!

**Comment by jycouet** [comment]
@frederikhors you should check the code now.
I think that it's 🥳🥳🥳, 

I didn't update the `Demo 1`. I let you do it? (Without tailwind, With `<KitQLInfo ...>`)

Let me know, if not I will do the update.

**Comment by jycouet** [comment]
FYI: https://twitter.com/jycouet/status/1506253283526156292?s=20&t=d__jm433J23wr6YRBEdVSw

**Review by frederikhors — APPROVED** [review]

**Comment by frederikhors** [comment]
Merged. If I remove Tailwind you'll switch demo 1 with this?

**Comment by jycouet** [comment]
Yes, I will 👍
(I will review as well :p)

**Comment by jycouet** [comment]
I let you proceed @frederikhors 
I just pushed a new version and I would like it to be synchronized asap.

Thx

**Comment by frederikhors** [comment]
I updated all! I can PR on your project now.

**Comment by frederikhors** [comment]
Looking back, I'd rather you figure out where to put the code in your examples dir.

**Comment by jycouet** [comment]
You fork my repo, and change demo 1 by this demo without tailwind?
Then PR my repo.

Like this, we will see changes 👍
Let me know, if not I'll do it.

**Pull request merged**